### ISSUE TITLE
Printing inconsistent width characters and spacing

### ISSUE THREAD

**Issue by leighmcculloch** [stellar/stellar-cli#1568]
### What version are you using?

21.4.1

### What did you do?

<img width="1011" alt="Screenshot 2024-08-28 at 3 38 32 PM" src="https://github.com/user-attachments/assets/eaf2a9d8-85c1-4db9-a3ea-51fb68110fb1">

### What did you expect to see?

Consistent spacing around emojis.

### What did you see instead?

Inconsistent spacing around emojis.

cc @fnando @elizabethengelman

**Comment by leighmcculloch** [comment]
I remember this when I added emojis to the snapshot create command, there were some emojis that were two code points wide and the terminals I tested in that affected the spacing against following characters. In that command I compensated by adding in an additional space after some emojis to have them align.

We need to check how this renders in a variety of terminals.

We may wish to limit our emojis to single codepoint emojis if we can't find common consistency.
